lib/register.coffee

```diff
@@ -15,37 +15,38 @@ limitations under the License.
 ###
 
 Promise = require('bluebird')
+postAsync = Promise.promisify(require('request').post, multiArgs: true)
 _ = require('lodash')
 
+randomHexChar = ->
+	return _.random(15).toString(16)
```

Comment:
Right, but before it was used only for  uuids and not apikeys, so the need for being crypto. secure wasn't as pressing... now it is.